modules/end-user-guide/examples/che-ref_a-kubernetes-secret-as-an-variable.adoc

```diff
@@ -54,10 +39,8 @@ metadata:
 data:
   mykey: myvalue
 ----
-+
-This results in the environment variable named `FOO_ENV` and the value `myvalue` being provisioned into all workspace containers.
-+
-If the Secret provides more than one key, the environment variable name must be provided for each of the data keys as follows:
+
+* If the secret provides more than one data key, each key must have its own environment variable name declared in the `annotations`. In the next example, the two environment variables `FOO_ENV` and `OTHER_ENV` and the two `data` values `myvalue` and `othervalue` will be provisioned into all workspace containers.
```

Comment:
```suggestion
* If the secret provides more than one data key, declare an environment variable name for each key in the `annotations`. The following example provisions the `FOO_ENV` and `OTHER_ENV` environment variables and the `myvalue` and `othervalue` data values into all workspace containers.
```

Reply:
I wonder if there's a convention/dependency here, like so:
`<mykey>_env-name` and `<mykey>`
`<otherkey>_env-name` and `<otherkey>`

Reply:
What about this?

If the secret has more than one data key, declare an environment variable name using a `<_key_>_env-name` annotation for each key, as shown in the following example. In this example, you provision the `myvalue` and `othervalue` data key values, and the environment variable names, `FOO_ENV` and `OTHER_ENV`, into all workspace containers.